specs/PermissionManagement.md

```diff
@@ -503,6 +510,13 @@ namespace Microsoft.Web.WebView2.Core
         // developers use the [Local Font Access API](https://wicg.github.io/local-font-access/)
         // to query the system fonts available for styling web content.
         LocalFonts = 10,
+
+
+        // Indicates permission for a site to use system exlusive messages to access
+        // Musical Instrument Digital Interface (MIDI) devices. Permission is requested
+        // when developers use the [Web MIDI API](https://developer.mozilla.org/en-US/docs/Web/API/Web_MIDI_API)
+        // to request MIDI access.
```

Comment:
I assume based on the enum name that this is only when using the MIDI API to request system exclusive MIDI access and not otherwise?

```suggestion
        // to request system exclusive MIDI access.
```